src/services/scheduler.ts

```diff
@@ -16,6 +17,11 @@ const workerShutdownEmitter = new EventEmitter();
 export async function start(dbId: string, dbUser:string) {
   // use the same connection for the scheduler and its operations
   const conn = await TypeormWrapper.createConn(dbId, { name: `${dbId}-${Math.floor(Math.random()*10000)}-scheduler`, });
+  // create a dblink server for SP. used to reuse connections
+  // https://aws.amazon.com/blogs/database/migrating-oracle-autonomous-transactions-to-postgresql/
+  await conn.query(`CREATE EXTENSION IF NOT EXISTS dblink;`);
+  await conn.query(`CREATE SERVER IF NOT EXISTS loopback_dblink FOREIGN DATA WRAPPER dblink_fdw OPTIONS (host '${config.dbHost}', dbname '${dbId}', port '${config.dbPort}');`);
```

Comment:
Quick question: Is this "server" entity scoped per database? If not we'll have issues with multitenancy and maybe we should include the `${dbId}` in the name of the link?

Reply:
oh hmmm I guess the tests don't stress that either because they all run independently and create a single db. let me create 2 dbs in one test

Reply:
good question

Reply:
yep the test failed, fixing it